extensions/amp-lightbox-viewer/0.1/service/lightbox-manager-impl.js

```diff
@@ -41,18 +47,19 @@ export class LightboxManager {
     /** @const @private {!../../../../src/service/ampdoc-impl.AmpDoc} */
     this.ampdoc_ = ampdoc;
 
-    /**
-     * Ordered list of lightboxable elements
-     * @private {?Array<!Element>}
-     **/
-    this.elements_ = null;
-
     /**
      * Cache for the `maybeInit()` call.
      * @private {?Promise}
      **/
     this.initPromise_ = null;
 
+    /**
```

Comment:
Basic idea in this implementation is to put lightbox group id --> elements in a map. Each lightbox group gets its own carousel and gallery.